src/main/proto/stremio/core/models/catalogs_with_extra.proto

```diff
@@ -30,8 +31,9 @@ message LoadablePage {
     Error error = 4;
     Page ready = 5;
   }
+  required stremio.core.deep_links.DiscoverDeepLinks deep_links = 6;
```

Comment:
tbh, I'd just keep this type here, rather than separately, since it's only going to be used here.

Reply:
It will probably be used in `catalogs_with_extra` too for search

Reply:
It can also be used in `catalog_with_filters` for the filters